providers/baseprov.c

```diff
@@ -101,6 +107,8 @@ static const OSSL_ALGORITHM *base_query(void *provctx, int operation_id,
         return base_decoder;
     case OSSL_OP_STORE:
         return base_store;
+    case OSSL_OP_RAND:
```

Comment:
Does this mean that we cant load the fips provider without also having the base provider present?

Reply:
No.  An old FIPS provider will use the old calls and work.  A more recent FIPS provider will use the new calls and fall back to the old & should work.

Despite that, it's difficult to see how a FIPS provider would run without base/default as well.  Loading key material becomes impossible without them.

Reply:
Yeah but it is possible..